contents/staticAnalysis/PMD.md

```diff
@@ -30,16 +30,26 @@ PMD doesn't support checking rules sets for the following languages. Only Copy/P
 ### Limitation
 Limitations are almost the same as [CheckStyle](checkStyle.md).
 
-- The examined code have to be compilable. The reason is described in [How does it work](#how-does-it-work) section.
-- Files will be examined one by one, there is no check to check multiple files at the same time (e.g. you cannot determine the full inheritance hierarchy of a class)
+- The examined code has to be compilable. The reason is described in [How does it work](#how-does-it-work) section.
+- Files will be examined one by one. Therefore, there is no way to check multiple files at the same time. For example, you cannot determine the full inheritance hierarchy of a class as you need to examine the parent class while checking the child class.
```

Comment:
"Files will be examined one by one. Therefore, there is no way to check multiple files at the same time." I think you just mention one enough, since one implies the other.